src/copyfiles.bat

```diff
@@ -14,22 +14,22 @@ if %PLATFORM%==x64   set PLATFORM_X=x64
 
 set DBGHELPERS=dbghelp_%PLATFORM_X%
 copy /y %DBGHELPERS%\*.* %DEST%
-if errorlevel 1 exit /b 1
+if errorlevel 1 ((@ECHO Failed to copy dbghelp_*) & (exit /b 1))

```

Comment:
These messages are only partially helpful. A better way to debug batch files is to remove the `@echo off` from the top, so I think these additions are not worth their weight.

Reply:
An alternate philosophy is to provide some meaningful information when there's a failure so that you don't have to re-run anything (potentially changing the conditions under which the failure originally occurred) to get more details.

Reply:
Good point, though the build script tries to be idempotent and only do what is needed (though it mostly delegates this part to MS Build).

Alternatively to defensive programming perhaps it would make sense to directly address the cause of the failure you were seeing?

Reply:
The first problem is that I didn't know which file was failing to be copied. The script just failed quietly (`exit /b 1`). The error message from cmd/COPY didn't include details. I realize this is not the script's problem, but MS isn't likely to fix it.

Reply:
So which file was it? Copying a file implies a precondition that the file must exist, and if execution of the script can reach this point with the file not existing, then that points to a bug somewhere earlier in the script.

Reply:
It was one of the files created from the drmingw build. To anyone unacquainted with the innards of this part of the project, it seems like adding a specific note would at least enable them to look at the part of the script where the error message came from and more quickly drill down to the source of the problem.

Why is this code in the script?

    if errorlevel 1 echo copyfiles.bat: Warning: couldn't copy the C/C++ runtime DLLs

I consider it sensible, but I didn't add it. It very clearly conveys the source of the problem.

Reply:
> It was one of the files created from the drmingw build.

Okay, that makes sense. It's a separate project and may change beyond our control.

> Why is this code in the script?

Because it also copies files from a location beyond our control. Unfortunately I don't remember why it's merely a warning.